src/templates/index.html

```diff
@@ -1,3 +1,28 @@
-<html>
-    <h1> Hihihihihih hhohohoh </h1>
-</html>
\ No newline at end of file
+{% extends 'base.html' %}
+{% block content %}
+<div class="container">
+  <div class="row">
+    <div class="col-6">
+      <a href={{ url_for('subscribe') }}><button type="button"  id='updateme' class="btn btn-lg btn-primary btn-dark">Update Me -></button></a>
+      <h1 id="title"> Festify</h1>
+      <p>Never miss a festival ticket sale.<br></p>
+      <p>Never miss the time of your life.</p>
+      <h5>30.05.2021</h5>
+      <p>Last update check</p>
+    </div>
+    <div class="col-2">
+      <img src="../static/images/Awakenings.png">
+      <p>Last update: <br> 05.05.2021</p>
+    </div>
+    <div class="col-2">
+      <img src="../static/images/Dekmantel.png">
+      <p>Last change: <br> {{events[46].lastChange}}</p>
```

Comment:
This part of repeating the same code for each event should not be done manually, since you don't know form before how many events are there and their key could change. For example this doesn't run for me since the db keys are different in mine.

In Flask templates you can use for loops as you would do in python. In every iteration you create an html element for each of the events in the list, something like:
`{%for event in events%}  
    <div>{{event.name}}</li>
{%endfor%}`